comms/src/lib.rs

```diff
@@ -1,5 +1,15 @@
+#[macro_use]
+extern crate log;
+
+#[macro_use]
+mod macros;
+
+#[macro_use]
```

Comment:
Dont think these are all required

Reply:
They are needed to use macros within the crate - the extern is not needed, I've added `use log::*` in the modules which use the log macros - thanks!